flowTypes.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+export type ConfigureParams = {
```

Comment:
I think we can make it work so that types can be imported from `react-native-google-signin` directly. I'd take a look at how react-navigation does it: https://github.com/react-navigation/react-navigation/blob/master/flow/react-navigation.js

Reply:
yes that's right I forgot about Flow's module. I'll do that...